.circleci/real_config.yml

```diff
@@ -365,7 +365,9 @@ commands:
             fi
             echo <<parameters.extras-requires>> >> /tmp/cachefile
             if [ -n <<parameters.extra-requirements-file>> ]; then
-              cat <<parameters.extra-requirements-file>> >> /tmp/cachefile
+              for i in <<parameters.extra-requirements-file>>; do
+                cat $i >> /tmp/cachefile
+              done
```

Comment:
allow multiple requirements files to be specified;  we want multiple requirements.txt files for CI purposes, but not in the build